react/src/js/components/Consonant/Helpers/lana.js

```diff
@@ -0,0 +1,34 @@
+export const loadLana = (options = {}) => {
+    if (window.lana) return;
+
+    const lanaError = (e) => {
+        if (window.lana) {
+            window.lana.log(e.reason || e.error || e.message, { errorType: 'i' });
+        }
+    };
+
+    window.lana = {
+        log: async (...args) => {
+            window.removeEventListener('error', lanaError);
+            window.removeEventListener('unhandledrejection', lanaError);
+            // eslint-disable-next-line import/no-unresolved, import/extensions
+            await fetch('https://milo.adobe.com/libs/utils/lana.js');
```

Comment:
This should load from `www.adobe.com/libs/utils/lana.js` instead of milo.adobe.com

Reply:
ok updated